src/components/LooMap/Markers.tsx

```diff
@@ -102,13 +102,33 @@ const MarkerGroup: React.FC<{
 }> = ({ geohash, maxClusterRadius }) => {
   const router = useRouter();
   const [mapState, setMapState] = useMapState();
+  const [lastContribution, setLastContribution] = useState<
+    number | undefined
+  >();
   const map = useMap();
 
   const { appliedFilters: filters } = mapState;
 
+  useEffect(() => {
```

Comment:
we use this information to detemine how long it's been since the last contribution the user has made. If it hasn't been too long since then, we want to pass `invalidateCache: true` so that their updates are reflected on everybodys' map.